src/classes/UTIL_Http_TEST.cls

```diff
@@ -307,4 +307,13 @@ public with sharing class UTIL_Http_TEST {
         Test.setMock(HttpCalloutMock.class, callout);
     }
 
+    public static void mockChargeCaptureGroup(String responseBody){
+        CalloutMock callout = new CalloutMock()
+                .withStatusCode(UTIL_Http.STATUS_CODE_OK)
+                .withStatus(UTIL_Http.STATUS_OK)
+                .withBody(responseBody);
+
+        Test.setMock(HttpCalloutMock.class, callout);
+    }
```

Comment:
I would move this mock into the NPSP_Batch_TEST file also, since that is the only place it is used currently and it will make that test easier to read and have one less dependency for that class (higher cohesion, looser coupling).